reinforcement_learning/README.md

```diff
@@ -0,0 +1,93 @@
+# Reinforcement Learning
+
+This package contains all the resources to train a dialogue policy using reinforcement learning. It comprises:
+
+  * Gymnasium environments
+  * Training scripts
+
+## Gymnasium environments
+
+To use the environments, you first need to register them with Gymnasium, to do so you need to execute their attached script.
+For example, to register the `DialogueEnvMovieBot-v0` environment, you need to execute the following command:
+
+```bash
+python -m reinforcement_learning.environment.dialogue_env_moviebot
+```
+
+Note that to use the environments you will need to import them from the `reinforcement_learning.environment` package, for example:
+
+```python
+from reinforcement_learning.environment.dialogue_env_moviebot import DialogueEnvMovieBot
+```
+
+## Training scripts
+
+Currently, there are two reinforcement learning algorithms supported:
+
+  * Deep Q-Learning
+  * Advantage Actor-Critic
+
+Each algorithm has its own trainer, which can be found in the `reinforcement_learning.training` package.
+
+Before training a dialogue policy, you need to prepare a configuration file.
+A default configuration for each supported algorithm is provided in the `config/rl` folder.
+
+A dialogue policy is trained by executing the following command:
+
+```bash
+python -m reinforcement_learning.train_dialogue_policy -c <training_configuration>
+```
+
+The training logs are stored in Weight and Biases.
```

Comment:
Maybe mention that users need to configure W&B to use this option.

Reply:
Done, I also added a link to W&B documentation if needed.